app/models/episode_image.rb

```diff
@@ -3,8 +3,12 @@ class EpisodeImage < ApplicationRecord
 
   belongs_to :episode, touch: true, optional: true
 
-  validates :height, :width, numericality: {greater_than_or_equal_to: 1400, less_than_or_equal_to: 3000}, if: :status_complete?
-  validates :width, comparison: {equal_to: :height}, if: :status_complete?
+  validates :height, :width, numericality: {greater_than_or_equal_to: 1400, less_than_or_equal_to: 3000}, if: :should_validate?
+  validates :width, comparison: {equal_to: :height}, if: :should_validate?
+
+  def should_validate?
+    status_complete? && (height_changed? || width_changed?)
```

Comment:
Any case for moving this to the shared `ImageFile`?  `ITunesImage` has similar validations, and there's a `:format` validation as well.

I see 1829 invalid episode images in prod ... as recently created as Feb 2023.  Bit surprised by that.

Reply:
image format on this one wasn't an issue, just the dimensions; probably we enforced a more lax value in the past.

Reply:
Just checked prod, and I see 1829 invalid EpisodeImages.  But 0 ITunesImages.  Since this isn't a problem there, probably don't need the validation in both models.